cmd/crowdsec-cli/alerts.go

```diff
@@ -0,0 +1,127 @@
+package main
+
+import (
+	"context"
+	"net/url"
+
+	"github.com/crowdsecurity/crowdsec/pkg/apiclient"
+	"github.com/crowdsecurity/crowdsec/pkg/cwhub"
+	"github.com/go-openapi/strfmt"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+var Scenario, AlertID, IP, Range, Since, Until, Source string
+var ActiveDecision bool
+
+func NewAlertsCmd() *cobra.Command {
+	/* ---- ALERTS COMMAND */
+	var cmdAlerts = &cobra.Command{
+		Use:   "alerts [action]",
+		Short: "Manage alerts",
+		Long: `
+Alerts Management.
+
+To list/add/delete decisions
+`,
+		Example: `cscli alerts [action] [filter]`,
+		Args:    cobra.MinimumNArgs(1),
+		PersistentPreRun: func(cmd *cobra.Command, args []string) {
+			var err error
+			if csConfig.LapiClient == nil {
+				log.Fatalln("There is no configuration on 'api_client:'")
+			}
+			apiclient.BaseURL, err = url.Parse(csConfig.LapiClient.Credentials.Url)
+			if err != nil {
+				log.Fatalf("failed to parse Local API URL %s : %v ", csConfig.LapiClient.Credentials.Url, err.Error())
+			}
+
+			//initialize cwhub
+			if err := cwhub.GetHubIdx(csConfig.Cscli); err != nil {
+				log.Fatalf("Failed to load hub index : %s", err)
+			}
+
+			scenarios := []string{}
+			for _, scenario := range cwhub.GetItemMap(cwhub.SCENARIOS) {
```

Comment:
should use the function that was added to cwhub : GetUpstreamInstalledScenariosAsString

Reply:
sure, but the feature is not there yet. I suggest to merge this before, and add GetUpstreamInstalledScenariosAsString in an other PR afterwards.